torchgeo/models/swin.py

```diff
@@ -31,17 +29,17 @@
     [255.0, 255.0, 255.0, 8160.0, 8160.0, 8160.0, 8160.0, 8160.0, 8160.0]
 )  # noqa: E501
 _mean = torch.zeros_like(_std)
-_sentinel2_ms_satlas_transforms = AugmentationSequential(
+_sentinel2_ms_satlas_transforms = K.AugmentationSequential(
     K.Normalize(mean=_mean, std=_std),
-    Lambda(lambda x: torch.clamp(x, min=0.0, max=1.0)),
-    data_keys=["image"],
+    Lambda(lambda x: torch.clamp(x, min=0.0, max=1.0)),  # type: ignore[arg-type]
```

Comment:
I think we decided we're supposed to wrap Lambda in an ImageSequential in order to get the style tests to pass?